template/src/index.ts.ejs

```diff
@@ -11,7 +11,9 @@ import {
 export function <%- ex.name %>(): number {
   <% if (ex.input) { -%>
     <% if (isJsonEncoded(ex.input)) { -%>
-      <% if (isPrimitive(ex.input)) { -%>
+      <% if (isBuffer(ex.input)) { -%>
+        const input: <%- toTypeScriptType(ex.input) %> = Host.base64ToArrayBuffer(JSON.parse(Host.inputString()))
```

Comment:
Depending on if you think it'd be confusing for folks to get the `ArrayBuffer` handle instead of a typed array view, we may want to wrap the result of `base64ToArrayBuffer` in a `Uint8Array`.

```suggestion
        const input: <%- toTypeScriptType(ex.input) %> = new Uint8Array(Host.base64ToArrayBuffer(JSON.parse(Host.inputString())))
```

Reply:
hmm, i'm not sure. `input` here needs to be ArrayBufferLike. wouldn't that violate it? it should be the same type as Host.inputBytes()

Reply:
Hm, both `Uint8Array` and `ArrayBuffer` are `ArrayBufferLike` because they both match the `{byteLength: number, slice(begin: number, end: number): ArrayBuffer}` type.

Reply:
let's follow back up on this discussion. it feels like a broader change than just this PR. I'll file an issue.